### DISCUSSION TITLE
I simplified the example on the alpine website

### DISCUSSION THREAD

**Discussion by Jak2k** [alpinejs/alpine#4356] [2. Show and tell]
![image](https://github.com/user-attachments/assets/6ec26806-4fd1-439f-9080-135bd5070ff9)
(Maybe show how alpine can do something, html can't do.)

**Comment by ekwoka** [comment]
It's not a bad point, but I don't think there is much that you can demo in such a small amount of code that won't have a "sort of kind of the same" html only option.
